API/DTOs/ServiceDTO.cs

```diff
@@ -1,7 +1,18 @@
-﻿namespace API.DTOs
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace API.DTOs
 {
     public class ServiceDTO
     {
         public Guid? Id { get; set; }
+        [Required]
+        public string? Name { get; set; }
+        [Required]
+        public string? Description { get; set; }
+        [Required]
+        public decimal? Price { get; set; }
+        public int? DurationInMinutes { get; set; }
```

Comment:
vienas service gali tureti labai daug datu, kada ji galima uzsakyt. Tai reiktu kazkaip pridet lista is kazkokio objekto Slot, kuris nurodytu visas sito serviso vietas